<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qtBintree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>680</width>
      <height>660</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>45</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inser_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="insert_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Вставка узла</string>
    </property>
   </widget>
   <widget class="QLabel" name="insert_status_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>45</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="del_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="del_status_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="del_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Удаление узла</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="search_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>195</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="search_status_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>195</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>195</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>165</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Поиск элемента</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Балансировка дерева</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preOrder_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прямой обход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simOrder_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Симметричный обход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="postOrder_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обратный обход</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Order_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>285</y>
      <width>371</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="example_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пример дерева</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_tree_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить дерево</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="print_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>140</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="vprint_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вертикальная печать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hprint_btn_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Горизонтальная печать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
